src/main/groovy/com/wikia/gradle/marathon/ConfirmationTask.groovy

```diff
@@ -10,8 +10,7 @@ class ConfirmationTask extends DefaultTask {
     def confirmation() {
         def console = System.console()
         if (console != null) {
-            println "CONFIRMATION: type project name (" + project.name + ")";
-            def line = console.readLine();
+            def line = console.readLine("\nCONFIRMATION: type project name (" + project.name + ")");
```

Comment:
I'm piggybacking this change in to fix the prompt display when deploying, it currently looks like:

```
service:user-preference:deployConfirmation
CONFIRMATION: type project name (user-preference)
> Building 0% > :service:user-preference:deployConfirmationuser-preference
```

But it should be formatted like:
```
> Building 0% > :tools:create-new-service:run
new service name: my-new-service
```

This change makes the prompt identical to [create-new-service](https://github.com/Wikia/pandora/blob/master/tools/create-new-service/build.gradle#L40)